releases/release-1.31/release-notes/maps/pr-118148-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 118148
+releasenote:
+  text: The kube-scheduler exposes "/livez" and "/readz" for health checks that are
```

Do we want to use past tense here?
```suggestion
  text: Exposed the kube-scheduler "/livez" and "/readz" endpoints for health checks that are
```

---

Yep, I think that's a good change- the [PR](https://github.com/kubernetes/kubernetes/pull/118148) specifically added those endpoints so I think it would be good to make it past tense here.